packages/shared/src/__tests__/__snapshots__/DropMarker.test.js.snap

```diff
@@ -1,31 +1,167 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`calculates the offsets base on icon size 1`] = `
-<Icon
-  color="#00a991"
-  name="place"
-  size={50}
+<Component
   style={
     Object {
-      "left": -25,
+      "backgroundColor": "#fff",
+      "borderColor": "#7f91a8",
+      "borderRadius": 4,
+      "borderWidth": 0.5,
+      "height": 24,
+      "padding": 2,
       "position": "absolute",
-      "top": -50,
+      "start": -12,
+      "top": -24,
+      "width": 24,
     }
   }
-/>
+>
+  <Component
+    style={
+      Object {
+        "backgroundColor": "#0176D2",
+        "borderRadius": 2,
+        "flex": 1,
+      }
+    }
+  >
+    <TextIcon
+      code=""
```

Comment:
Is this going to be any different when we change the icon? It's not crucial but maybe this would not detect a new icon.

Reply:
I fear that there is very little we can do about this.